stable/kiam/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: kiam
-version: 2.0.1-rc3
-appVersion: 3.0-rc1
+version: 2.0.1-rc4
```

Comment:
Could you please change `version` to `2.1.0` as we're no longer using an RC version of the image?

Reply:
@ewbankkit done!  thanks for your attention.